images/origin/system-container/config.json.template

```diff
@@ -70,15 +70,13 @@
 	    "source": "proc"
 	},
 	{
-	    "destination": "/dev",
-	    "type": "tmpfs",
-	    "source": "tmpfs",
-	    "options": [
-		"nosuid",
-		"strictatime",
-		"mode=755",
-		"size=65536k"
-	    ]
+		"destination": "/dev",
+		"type": "bind",
+		"source": "/dev",
+		"options": [
+			"rbind",
+			"rprivate"
```

Comment:
I will test on our cluster, it might actually work better, since I did see some strange log messages when EBS volumes were attached on the nodes for PVC's (which adds block devices to /dev).